core/src/i18n/en.yaml

```diff
@@ -896,6 +897,7 @@ git-repositories:
     secret: The name of the Secret with credentials used by the Function Controller to authenticate to the Git repository in order to fetch the Function's source code and dependencies.
     url: URL must be a valid Git repository address that starts with the "http(s)", "git", or "ssh" prefix.
 instances:
+  description: <0>Service Instance</0> makes the functionality of a service available for consumption.
```

Comment:
... or: `<0>Service Instance</0> is a provisioned instance of a service that can be consumed by an application. `